src/Reflex/Dom/SemanticUI/Button.hs

```diff
@@ -124,14 +137,16 @@ uiButtonAttrs UiButton{..} = T.unwords $ catMaybes
 -- provided by other functions such as 'uiButtonAnimated'.
 uiButton
     :: MonadWidget t m
-    => Dynamic t UiButton
+    => UiButtonType
```

Comment:
True.  Then I think if we're making a breaking change we should go for a more general one:

```
uiButton :: MonadWidget t m => UiButtonStatic -> Dynamic t UiButton ...
```

...and make UiButtonType be a field of UiButtonStatic.